firebase_admin/db.py

```diff
@@ -686,6 +686,7 @@ def __init__(self, credential, base_url, auth_override=_DEFAULT_AUTH_OVERRIDE):
           auth_override: A dictionary representing auth variable overrides or None (optional).
               Default value provides admin privileges. A None value here provides un-authenticated
               guest privileges.
+          timeout: HTTP request timeout in seconds (optional).
```

Comment:
What is the behavior if one isn't specified?

Reply:
In that case we don't set any timeout on the connection, which activates the defaults of the `requests` library. However, their default is essentially "never timeout". Would it be better to default to something like 60 seconds instead?

Reply:
I think we should document the default behavior.  If the old behavior was "never timeout" we should probably leave it be, in case anybody depended on it.

Reply:
Done